src/main/java/com/minecolonies/structures/event/RenderEventHandler.java

```diff
@@ -1,8 +1,10 @@
 package com.minecolonies.structures.event;
 
 import com.minecolonies.api.util.BlockUtils;
+import com.minecolonies.structures.client.StructureClientHandler;
 import com.minecolonies.structures.helpers.Settings;
 import com.minecolonies.structures.helpers.Structure;
+import com.sun.jna.StructureReadContext;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use classes from the Java API instead of Sun classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1191)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.sun.jna.StructureReadContext'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)